app/src/main/java/gov/hhs/cdc/trustedintermediary/etor/EtorDomainRegistration.java

```diff
@@ -65,6 +67,18 @@ public String openApiSpecification() {
 
     DomainResponse handleOrder(DomainRequest request) {
 
+        // Call the DemographicsRequestValidator class
+        DemographicsRequestValidator validator = new DemographicsRequestValidator(request);
+
+        // Validate token
+        try {
+            if (!validator.isValidToken()) {
```

Comment:
demographics unit test fails here, due to logger being null.

Reply:
This has been resolved in the follow-on commits.